src/main/scala/it/pagopa/interop/authorizationprocess/service/impl/AuthorizationManagementServiceImpl.scala

```diff
@@ -70,22 +70,39 @@ final case class AuthorizationManagementServiceImpl(
           case err: ApiError[_] if err.code == 404 => Future.failed(ClientNotFound(clientId))
         }
     }
+
+  override def updateKey(clientId: UUID, keyId: String, userId: UUID)(implicit
+    contexts: Seq[(String, String)]
+  ): Future[Unit] = withHeaders[Unit] { (bearerToken, correlationId, ip) =>
+    val request: ApiRequest[Unit] =
+      keyApi.updateKey(xCorrelationId = correlationId, clientId, keyId, UserSeed(userId), xForwardedFor = ip)(
+        BearerToken(bearerToken)
+      )
+    invoker.invoke(request, "Operator addition to client")
```

Comment:
I know this will not be merged, but comprehensive logs will help us during the run
```suggestion
    invoker.invoke(request, "Key user migration")
```